browser/css/menubar.css

```diff
@@ -63,14 +63,60 @@
 	display: none;
 }
 .main-nav {
-	height: 32px; /* on mouseover menubar items, border emerges */
+	height: 36px; /* on mouseover menubar items, border emerges */
```

Comment:
I think this is not needed 5 days ago on of my PRs from Nov. got merged that modifies this: (more info: https://github.com/CollaboraOnline/online/pull/3713)